src/ontology/cl-edit.owl

```diff
@@ -32065,9 +32068,38 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4033067 "EFO:0005289")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1530/REP-16-0500") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033067 "MGC")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34198768") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033067 "mural GC")
 AnnotationAssertion(rdfs:label obo:CL_4033067 "mural granulosa cell")
-SubClassOf(obo:CL_4033067 obo:CL_0002174)
+SubClassOf(obo:CL_4033067 obo:CL_0000501)
 SubClassOf(obo:CL_4033067 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000992))
-SubClassOf(obo:CL_4033067 ObjectSomeValuesFrom(obo:RO_0002207 obo:CL_0000501))
+SubClassOf(obo:CL_4033067 ObjectSomeValuesFrom(obo:RO_0002207 obo:CL_4033084))
+
+# Class: obo:CL_4033083 (squamous granulosa cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28892263") obo:IAO_0000115 obo:CL_4033083 "A granulosa cell that has a squamous morphology and form a single layer around the oocyte in primordial follicles.")
```

Comment:
I would suggest adding this " A granulosa cell that has a squamous morphology and form a single layer around the oocyte in primordial follicles. This cell develops directly into a cuboidal granulosa cell during the primordial-to-primary follicle transition."